src/installation.rst

```diff
@@ -3,64 +3,53 @@
 Installation
 ============
 
-.. warning::
-
-   Cylc 8.0b3 is the final *beta release* of Cylc 8, a major upgrade from Cylc 7.
+.. note::
 
-   All Cylc 8 system components are in place but not yet heavily tested by users. 
-   Cylc 7 is still available if needed.
+   Cylc |release| is a pre release not recommended for production use.
 
-   This documentation has been revised for Cylc 8, but some new features are
-   yet to be fully described.
+   Please report any issues to the `Cylc 8 Discourse Support Group`_.
 
 
 Quick Installation
 ------------------
 
 Cylc runs on Unix systems including Linux and Mac OS.
 
-.. highlight:: sub
-
-.. list-table::
-   :class: grid-table
-
-   * - .. rubric:: Via Conda (recommended):
-     - .. rubric:: Via Pip (+npm):
-
-   * - ::
+Via Conda (recommended)
+^^^^^^^^^^^^^^^^^^^^^^^
 
-          $ conda install cylc-flow
+.. code-block:: sub
```

Comment:
What is this lexer?

Reply:
https://cylc.github.io/cylc-sphinx-extensions/extensions/cylc.sphinx_ext.sub_lang.html

(simple lexer that is less likely to produce unexpected highlighting)